third-party/build.gradle

```diff
@@ -92,7 +148,7 @@ tasks.register('prepareBndFile') {
                 jar.entries().each { entry ->
                     if (entry.name.endsWith('.class')) {
                         def packageName = entry.name.replaceAll('/[^/]+$', '').replace('/', '.')
```

Comment:
needs a comment to explain what this exclude list is and what the regex is doing

Reply:
Josh is adding a comment